modules/nodes-containers-sysctls-setting.adoc

```diff
@@ -47,18 +40,16 @@ spec:
       value: "1024"
     - name: kernel.msgmax
       value: "65536"
-  ...
 ----
 
 . Create the pod:
 +
 [source,terminal]
 ----
-$ oc apply -f <file-name>.yaml
+$ oc apply -f sysctl-example-unsafe.yaml
```

Comment:
Same as above. If I apply this yaml then I receive following message:
```
W0616 20:41:43.345471 1397301 warnings.go:70] would violate PodSecurity "restricted:latest": forbidden sysctls (net.core.somaxconn, kernel.msgmax), allowPrivilegeEscalation != false (container "podexample" must set securityContext.allowPrivilegeEscalation=false), unrestricted capabilities (container "podexample" must set securityContext.capabilities.drop=["ALL"]), runAsNonRoot != true (pod or container "podexample" must set securityContext.runAsNonRoot=true), seccompProfile (pod or container "podexample" must set securityContext.seccompProfile.type to "RuntimeDefault" or "Localhost")
pod/sysctl-example-unsafe created
```
When I apply the yaml below:
```
apiVersion: v1
kind: Pod
metadata:
  name: sysctl-example-unsafe
  namespace: default
spec:
  containers:
  - name: podexample
    image: centos
    command: ["bin/bash", "-c", "sleep INF"]
    securityContext:
      runAsUser: 2000
      runAsGroup: 3000
      allowPrivilegeEscalation: false
      capabilities:
        drop: ["ALL"]
  securityContext:
    runAsNonRoot: true
    seccompProfile:
      type: RuntimeDefault
    sysctls:
    - name: kernel.shm_rmid_forced
      value: "0"
    - name: net.core.somaxconn
      value: "1024"
    - name: kernel.msgmax
      value: "65536"
```
I receive following message:
```
W0616 20:43:26.205690 1397980 warnings.go:70] would violate PodSecurity "restricted:latest": forbidden sysctls (net.core.somaxconn, kernel.msgmax)
pod/sysctl-example-unsafe created
```
The status of my pod is 
```
oc get pods
NAME                    READY   STATUS                   RESTARTS   AGE
sysctl-example-unsafe   0/1     ContainerStatusUnknown   0          67s
```